templates/vue-demo-store/pages/checkout/success/[id]/index.vue

```diff
@@ -63,10 +63,20 @@ const toggleView = () => {
   isExpand.value = !isExpand.value;
 };
 
-const formatDate = (date: Date | string) =>
-  new Date(date).toLocaleDateString(
-    (typeof navigator !== "undefined" && navigator.language) || "en-US",
-  );
+const { sessionContext } = useSessionContext();
+const navLang =
+  sessionContext.value?.languageInfo.localeCode ||
+  navigator?.language ||
+  "en-US";
+
+const formatDate = (date: Date | number) =>
```

Comment:
Why not as a helper function?

Reply:
as format is the biggest part of it, you can see exactly what will be displayed (here with hours and minutes for example). I'm okay with helper function for that but that was not the case for this fix